From: Guido Berhoerster <guido+debian@berhoerster.name>
Subject: Revert "qml: Use invisible components to get colors for ExtendedKeySelector"

This reverts commit 3f0d076a6e55551760a27b994de8aff16e6c0d.
This is a workaround until https://github.com/maliit/keyboard/issues/217 is
addressed by upstream.
---
 qml/keys/ExtendedKeysSelector.qml | 24 +++++-------------------
 1 file changed, 5 insertions(+), 19 deletions(-)

--- a/qml/keys/ExtendedKeysSelector.qml
+++ b/qml/keys/ExtendedKeysSelector.qml
@@ -92,17 +92,10 @@
 
         height: rowOfKeys.height
 
-        // Invisible tooltip to copy the qqc2 style colors from
-        ToolTip {
-            id: tip
-            visible: false
-        }
-
-        color: tip.background.color
         radius: Device.gu(0.8)
         border {
-            width: tip.background.border.width
-            color: tip.background.border.color
+            width: 1
+            color: "#888888"
         }
 
         onXChanged: {
@@ -144,21 +137,14 @@
                 property bool highlight: false
                 opacity: highlight ? 1.0 : 0.6
 
-                // Invisible text field to get selection colors from
-                TextField {
-                    id: textArea
-                    width: 0
-                    height: 0
-                    visible: false
-                }
-
-                Label {
+                Text {
                     id: textCell
                     anchors.centerIn: parent;
                     text: modelData
+                    font.family: Theme.fontFamily
                     font.pixelSize: fontSize
                     font.weight: Font.Light
-                    color: key.highlight ? textArea.selectionColor : textArea.color
+                    color: key.highlight ? Theme.selectionColor : Theme.fontColor
                 }
 
                 function commit(skipAutoCaps) {
